src/components/Search/Search.css

```
@@ -55,7 +54,8 @@
 }
 
 .Search--has-after .Search__input {
-  border-radius: 10px 0 0 10px;
+  border-radius: var(--vkui--size_border_radius--regular) 0 0
+    var(--vkui--size_border_radius--regular);
```

Учитывая, что уже есть `border-radius`, то мб просто обнулять?
```suggestion
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
```